Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -215,12 +215,29 @@ public Vector3 SliderTrackDirection
         public SliderEvent OnHoverExited = new SliderEvent();
         #endregion
 
-        #region Private Members
-        private float startSliderValue;
-        private Vector3 startPointerPosition;
-        private Vector3 startSliderPosition;
-        private IMixedRealityPointer activePointer;
-        private Vector3 sliderThumbOffset = Vector3.zero;
+        #region Protected Members
+        /// <summary>
+        /// Float value that holds the starting value of the slider.
+        /// </summary>
+        protected float startSliderValue;
```

Comment:
Could these be left [`private` with `protected` property accessors](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers)?